Sources/Orbit/Components/HorizontalScroll.swift

```diff
@@ -16,6 +16,15 @@ public enum HorizontalScrollItemHeight {
     case custom(CGFloat)
 }
 
+struct HorizontalScrollWidthKey: PreferenceKey {
```

Comment:
This looks like an implementation details, but it's sandwiched between `public` types.

Reply:
Where to put it though - public goes first - and they cannot be nested in the generic type.

Reply:
Where to put them though. They cannot go inside the component, so I at least ordered it by public first.

Reply:
Above previews?

Reply:
Updated - removed the specific key and unified with others.